src/unix/core.c

```diff
@@ -1215,6 +1223,13 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   pwd->shell = pwd->homedir + homedir_size;
   memcpy(pwd->shell, pw.pw_shell, shell_size);
 
+  /* Copy the gecos field */
+  pwd->gecos = pwd->shell + shell_size;
+  if (pw.pw_gecos != NULL) {
+    memcpy(pwd->gecos, pw.pw_gecos, gecos_size);
+  } else
+    *(pwd->gecos) = '\0';
```

Comment:
Yeah, the parens are a personal style habit, and I think the braces were required at some point.